src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -53,9 +53,8 @@ public class DriveTrain extends SubsystemBase {
     // used in lots of places, so create a local constant
     private static final double MAX_VELOCITY_METERS_PER_SECOND = FalconDriveController.MAX_VELOCITY_METERS_PER_SECOND;
 
-    // TODO get correct values
-    public static final double PATH_PLANNER_MAX_VELOCITY = 2.0; //5.0;
-    public static final double PATH_PLANNER_MAX_ACCELERATION = 2.0; //5.0;
+    public static final double PATH_PLANNER_MAX_VELOCITY = 4.5; //5.0;
+    public static final double PATH_PLANNER_MAX_ACCELERATION = 3.5; //5.0;
```

Comment:
We need to test:
 A) What the max speed/acceleration that we're comfortable with, that give a meaningful time benefit 
and 
B) Should all paths use the same values from these constants, or is it helpful in some cases to use path-specific velocity/acceleration values?

Reply:
Agreed